components/org.wso2.carbon.identity.api.server.claim.management/org.wso2.carbon.identity.rest.api.server.claim.management.v1/src/main/java/org/wso2/carbon/identity/rest/api/server/claim/management/v1/core/ServerClaimManagementService.java

```diff
@@ -969,6 +969,13 @@ private void validateAttributeMappings(List<AttributeMappingDTO> attributeMappin
                 throw handleClaimManagementClientError(ERROR_CODE_EMPTY_MAPPED_ATTRIBUTES_IN_LOCAL_CLAIM,
                         BAD_REQUEST, attributeMappingDTO.getUserstore());
             }
+            if (IdentityUtil.getHostName().contains("asgardeo.io") || IdentityUtil.getHostName().contains("asg.io")) {
```

Comment:
If we are whitelisting any hostnames this is not a very good way to do it.  It should be configurable if the list can be dynamic, and if they are static we can define constants and use.

Reply:
Fixe with https://github.com/wso2/identity-api-server/pull/440/commits/876e09128a4c2ca3343d6f9f215677934f5eadad where I have introduced a new config to check if we should use only the primary user store to save local claims